### DISCUSSION TITLE
Can I import from SteamDesktopAuthenticator (SDA)?

### DISCUSSION THREAD

**Discussion by n0pe999** [dyc3/steamguard-cli#424] [Q&A] [answered]
### Description

is it possible to import my files from sda to steamguard-cli? without a trade banned. and i want to ask is this legit also just making it sure.

**Comment by dyc3** [comment, marked as answer]
Yes! Simply point `steamguard-cli` to your `maFiles` from SDA. They will be automatically upgraded to be compatible with `steamguard-cli`.

See the quickstart: https://github.com/dyc3/steamguard-cli/blob/master/docs/quickstart.md#importing-existing-mafiles-from-steam-desktop-authenticator

**Reply by n0pe999** [reply]
can i see a legitimate that u are legit pls?

**Reply by dyc3** [reply]
I have no idea what you are asking for.

**Reply by n0pe999** [reply]
i mean im asking if u are legit or not because maybe u gonna steal my steam account etc

**Reply by dyc3** [reply]
No, I'm not going to steal your steam account. I just wrote the software. As long as you download `steamguard-cli` from this repo's releases or compile it from the source code from here, it's safe.

**Reply by n0pe999** [reply]
ok thanks ill read the quickstart
